specification/trafficmanager/resource-manager/Microsoft.Network/stable/2018-04-01/trafficmanager.json

```diff
@@ -0,0 +1,1578 @@
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Endpoint-PUT-External-WithGeoMapping": {
+            "$ref": "./examples/Endpoint-PUT-External-WithGeoMapping.json"
+          },
+          "Endpoint-PUT-External-WithLocation": {
+            "$ref": "./examples/Endpoint-PUT-External-WithLocation.json"
+          },
+          "Endpoint-PUT-External-WithSubnetMapping": {
+            "$ref": "./examples/Endpoint-PUT-External-WithSubnetMapping.json"
+          },
+          "Endpoint-PUT-External-WithCustomHeaders": {
+            "$ref": "./examples/Endpoint-PUT-External-WithCustomHeaders.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "Endpoints"
+        ],
+        "operationId": "Endpoints_Delete",
+        "description": "Deletes a Traffic Manager endpoint.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group containing the Traffic Manager endpoint to be deleted."
+          },
+          {
+            "name": "profileName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Traffic Manager profile."
+          },
+          {
+            "name": "endpointType",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The type of the Traffic Manager endpoint to be deleted."
+          },
+          {
+            "name": "endpointName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Traffic Manager endpoint to be deleted."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The Traffic Manager Endpoint was deleted successfully.",
+            "schema": {
+              "$ref": "#/definitions/DeleteOperationResult"
+            }
+          },
+          "204": {
+            "description": "The Traffic Manager Endpoint does not exist. It could have been deleted on a previous request."
+          },
+          "default": {
+            "description": "Default response. It will be deserialized as per the Error definition.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Endpoint-DELETE-External": {
+            "$ref": "./examples/Endpoint-DELETE-External.json"
+          }
+        }
+      }
+    },
+    "/providers/Microsoft.Network/checkTrafficManagerNameAvailability": {
+      "post": {
```

Comment:
This service endpoint does need have a subscriptionId/resourceGroup in the path?

Reply:
It does not need a subscription ID. This is a globally available query to find out if a DNS name has been reserved.

Reply:
Thanks @allencal is this service already in production? If not we need an ARM\API review here

Reply:
This service is already in production. Some new operations are being added for custom headers, custom subnets and MultiValue response.